C4SGWeb/src/app/about/about.component.html

```diff
@@ -2,9 +2,9 @@
   Our Office Location
 </b>
 
-<p>
-  todo add static google map here
-</p>
+<br>
+
+<img width="640" src="https://maps.googleapis.com/maps/api/staticmap?autoscale=1&size=640x420&maptype=roadmap&key=AIzaSyD9n6KasSkB0QvESMyH0tL8UhEtxjvEVrs&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xff0000%7Clabel:%7CUnited+States" alt="Code for Social Good Volunteer Locations">
```

Comment:
nice 👍 how about making it easy to add future markers? i.e. read a static array in the component and then create the url